omnigan/generator.py

```diff
@@ -120,8 +124,10 @@ def decode_tasks(self, z):
     def encode(self, x):
         return self.encoder.forward(x)
 
-    def forward(self, x, translator="f", classifier_probs=[1, 0, 0, 0]):
-        """Computes the translation of an image x to a flooding domain
+    def forward_x(self, x, translator="f"):
```

Comment:
this is the actual forward, name emphasizes it will infer conditioning information from the sole image, not using ground truth data from a batch (unlike `translate_batch`)